reference/onenote/notebook.md

```diff
@@ -119,6 +120,35 @@ OneNote.run(function (context) {
 }); 
 ```
 
+### getRestApiId()
+Gets the id that is compatible with the REST API.
+
+#### Syntax
+```js
+notebookObject.getRestApiId();
+```
+
+#### Parameters
+None
+
+#### Returns
+string
+
+#### Examples
+```js
+
+OneNote.run(function(ctx){
+    // Get the current notebook.         
+    var notebook = ctx.application.getActiveNotebook();
+    var restApiId = notebook.getRestApiId();
```

Comment:
can you add a line for using baseUrl property for SharePoint? add comment to say needed for SharePoint and perhaps a link to our REST API doc?